Frontend/node/default.nix

```diff
@@ -0,0 +1,50 @@
+{ inputs, lib, flake-parts-lib, ... }:
+let
+  inherit (flake-parts-lib)
+    mkPerSystemOption;
+  inherit (lib)
+    types;
+in
+{
+  options = {
+    perSystem = mkPerSystemOption
+      ({ config, self', lib, pkgs, system, ... }:
+        let
+          nodePackages = inputs.dream2nix.lib.makeFlakeOutputs {
+            systems = [ system ];
+            source = ./.;
+            projects = {
+              "atlas-ui" = {
+                name = "atlas-ui";
```

Comment:
Why `atlas-ui` (as naming)?

Reply:
that was the package name in ui-common from before. It can be whatever, happy to change it to common.

Reply:
I'm trying to understand the organization, but I don't think I get it yet. 

Is `Frontend/node` more or less independent from the rest of the project? Also there is a `watch.js` in this directory, yet `webpack-dev-server` is define *outside*. You want to keep the Nix alongside the code they are packaging generally.

Reply:
`webpack-dev-server` is specific to watch hence it living in `watch.nix`. Perhaps you would prefer `Frontend/nix/watch/default.nix` and `Frontend/nix/watch/watch.js`? Ultimately, I couldn't a good way to split the watch/bundle/node apart in a way that makes sense from every angle and so this is what I ended up with.

Reply:
I'd just put `watch.js` alongside `watch.nix` for now.

Still trying to wrap my head around the whole watch nix organization logic ...